crates/swc/tests/tsc-references/classes/propertyMemberDeclarations/constructorParameterShadowsOuterScopes/input.ts/es2015.1.normal/output.js

```diff
@@ -6,15 +6,15 @@ var x = 1;
 class C {
     constructor(x1){
         this.b // error, evaluated in scope of constructor, cannot reference x
-         = x1;
+         = x;
```

Comment:
This seems ok too as it would error. Original code was:

```ts
var x = 1;
class C {
    b = x; // error, evaluated in scope of constructor, cannot reference x
    constructor(x: string) {
        x = 2; // error, x is string
    }    
}
```